### DISCUSSION TITLE
tutorials/shaders/introduction_to_shaders

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#55] [User-contributed notes]
# tutorials/shaders/introduction_to_shaders

This page explains what shaders are and will give you an overview of how they work in Godot. For a detailed reference of the engine's shading language, see Shading language. Shaders are a special k...

https://docs.godotengine.org/en/stable/tutorials/shaders/introduction_to_shaders.html

**Comment by Xinart** [comment]
# What are Vertice and Vertex ?  
**Vertex** is a point where two straight lines meet.  

They're found in 2D and 3D objects where the sides or edges of these objects meet

![vertex explenation image](https://d138zd1ktt9iqe.cloudfront.net/media/seo_landing_files/vertex-of-angle-02-1648461511.png)

**Vertice** is the plural form of vertex.

**Reply by MonstrousSoftware** [reply]
Typo: the plural is "vertices"

**Reply by jduhking** [reply]
How do Vertex look like in 2D images?

**Reply by jduhking** [reply]
Specifically in pixel art

**Reply by Calinou** [reply]
2D sprites are drawn with four vertices only (one in each corner of the sprite). Most 2D elements are rectangles with transparent elements, so the actual shape you see won't necessarily be a rectangle.

For actual polygon-based 2D graphics in Godot, use the [custom drawing in 2D methods](https://docs.godotengine.org/en/latest/tutorials/2d/custom_drawing_in_2d.html), the Line2D or Polygon2D nodes.
